code/game/objects/items/devices/radio/radio_objects.dm

```diff
@@ -544,7 +552,10 @@ GLOBAL_LIST_EMPTY(deadsay_radio_systems)
 
 /obj/item/radio/examine(mob/user)
 	. = ..()
-	. += "<span class='notice'><b>Alt-Click</b> to toggle [src]'s hotmic!</span>"
+	if(!istype(src, /obj/item/radio/uplink))
+		. += "<span class='notice'><b>Alt-Click</b> to toggle [src]'s hotmic!</span>"
+		. += "<span class='notice'><b>Ctrl-Shift-Click</b> to toggle [src]'s speaker!</span>"
+
```

Comment:
I'm not quite sure what you're talking about, I don't need that information in /uplink, so I did a check - the code you provided below confuses me.

Reply:
Ah, sorry I didn't see the `!`. So actually, we want something like this:
```dm
/obj/item/radio/examine(mob/user)
	. = ..()
	. += show_examine_hotkeys()

/obj/item/radio/proc/show_examine_hotkeys()
	. += "<span class='notice'><b>Alt-Click</b> to toggle [src]'s hotmic!</span>"
	. += "<span class='notice'><b>Ctrl-Shift-Click</b> to toggle [src]'s speaker!</span>"

/obj/item/radio/uplink/show_hotkeys()
	return
```
That will display the hotkeys on regular radios but not uplinks.

Reply:
It inserts empty line in uplink, and i'm not sure how to fix it

Reply:
Ah right, I was able to fix it by doing `return list()` instead of just `return`. Also, forgot to include this, but you need `. = list()` at the start of `/obj/item/radio/proc/show_examine_hotkeys()` so the lines are properly separated. Here is my final code. feel free to give it a test. (Also, put the uplink proc in the uplink dm file of course)

![image](https://github.com/ParadiseSS13/Paradise/assets/42044220/6c18c547-6343-4fad-a4b5-e1c618c5c4bb)

Reply:
Yep, it works.
![image](https://github.com/ParadiseSS13/Paradise/assets/20109643/903223c9-9acb-4fee-be45-f33c06275fcc)